lib/entry/StateManager.js

```diff
@@ -157,5 +162,26 @@ module.exports = function(liveEntry) {
         return getLiveEntryServerNodeIdPromise;
     }
     
+    function isNewSession() {
+        let newSessionPromises = [];
+        newSessionPromises.push(sessionManager.isNewSession(liveEntry.entryId));
+        newSessionPromises.push(liveEntry.playlistGenerator.checkTranscodingProfileChange());
+
+        return Q.any(newSessionPromises)
```

Comment:
checkTranscodingProfileChange is sync. so instead of doing Q.any it should do_.every